### ISSUE TITLE
Hanging processes on windows 

### ISSUE THREAD

**Issue by kaddkaka** [wez/wezterm#857]
## Describe the bug
Some times when wezterm stops with ~"process terminated" and after closing the window with alt-f4, there is still some process running on window.

## Environment (please complete the following information):

 - OS: Windows 10 - version 1809
 - Version: wezterm 20210531-223926-20c4dcdc

## To Reproduce

1.  Start wezterm in ssh mode: `wezterm-gui.exe --ssh user@remote`
2.  Start a tmux session (not sure if needed)
3.  Let the connection drop (for example disconnecting wifi) (I had this triggered by power saving options)
     - Wait until the wezterm window is unresponsive
4.  Close the window with `alt-f4`

After this, the processes are still alive in task manager (2 for each instance closed this way). See picture.

## Configuration
No custom config file

## Expected behavior
The process should be killed completely and no longer take up system resources.

## Screenshots
![wez_hanged](https://user-images.githubusercontent.com/4508793/121008114-e8cf8980-c792-11eb-9aad-30afe2aa6caf.png)

**Comment by wez** [comment]
I couldn't trivially reproduce this.
Are you sure that those are wezterm-gui.exe processes?
Task Manager does some aggregation that can be misleading; make sure that you've expanded any chevron to the left, and then right click on the item(s) to call up the properties to confirm which process it is.

**Comment by kaddkaka** [comment]
Yes it seems to be wezterm-gui:

![image](https://user-images.githubusercontent.com/4508793/121083904-48a05180-c7e0-11eb-9e8c-b0618fb86d72.png)

**Comment by kaddkaka** [comment]
This still happens for me regularly. I added a sub-step under 3. in the description.

Is there some log or similar I can try to collect?

**Comment by wez** [comment]
I don't know if this was what you were seeing, but I pushed a fix for something I've seen a bit this past week.  Let me know if the next nightly build works for you (in ~30 mins from now).

**Comment by kaddkaka** [comment]
Today I found a hanged process again after the scenario described in this thread. I'm currently still running version `wezterm 20210718-221318-5904140a` however.

Has there been any updates in this area the last month? I will update version and get back with an update.

**Comment by kaddkaka** [comment]
I thought I saw some comment from someone regarding a fix for this, but I don't see it now. 

I updated my wezterm version yesterday, but I have seen hanging processes again, unfortunately. I currently don't have any ideas on how to troubleshoot it though. The amount of memory that the ghost processes use are in the 30- 170MB

![wez_hanged_2021_09_08](https://user-images.githubusercontent.com/4508793/132570504-0d8fee06-de13-4617-81fb-09fb695ea8d3.png)
